### PR TITLE
#2440 Introduce Quickstart VM

### PR DESCRIPTION
**Pull request by vitojeng**
required by #2440 

### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by chia7712** [comment]
@vitojeng how we verify this PR? Can I build quick vm on my local?

**Comment by vitojeng** [comment]
> @vitojeng how we verify this PR? Can I build quick vm on my local?

Yes, you can. 
Make sure you adready install [packer](https://www.packer.io/) in your local env.

For example of Ohara 0.7.1 quickstart VM, just type:
```
[quickstart]$ make OHARA_ENV=0.7.1 ova
```
Finally you can find a OVA file in the **build** directory.

**Comment by vitojeng** [comment]
Yes. I build the image in my ubuntu env, so far so good. ^^

I also add some helper task in Makefile to support the command line for testing or development.

**Comment by vitojeng** [comment]
Sorry, I misunderstan you message.
I will write a simple page for how to build.

**Comment by vitojeng** [comment]
@saivirtue Just add a simple document, please take a look. :)

File: docs/src/build_quickstart_vm.rst [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+.. _build_quickstart_vm:
+
+How to build Quickstart VM
+==========================
+
+Prerequisites
+-------------
+
+- Packer_: Packer is an open source tool for creating identical machine images
```
[the remark this answers is not in the document]
Comment (vitojeng) [reply docs/src/build_quickstart_vm.rst]: Thanks, already fixed.

**Pull request merged**